### PR TITLE
Path manager

### PR DESCRIPTION
**Pull request by TNicko**
## Description

This update introduces the Path Manager extension for Raycast, which allows users to easily manage their file system paths. It includes three main commands: Add Path, List Paths, and Open Path. Add Path lets users store a file path with an alias for later use. List Paths displays all saved paths with options to search, edit, or delete them. Open Path uses an alias to open a specified path in the terminal. Configuration settings support using either the default terminal or iTerm.

## Screencast

https://github.com/raycast/extensions/assets/92307192/b89da0e5-4ab0-4a27-b6f9-7970b006f014

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/path-manager/src/list-paths.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    await confirmAlert(options);
+  }
+
+  function handleAdd() {
+    push(<PathForm mode="edit" initialAlias={""} initialPath={""} />);
+  }
+
+  return (
+    <List
+      filtering={false}
```
Comment (pernielsentikaer) [inline extensions/path-manager/src/list-paths.tsx:135]: I can't search for the item I just created - maybe due to filtering=false

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- I'm getting this error when I load the extension for the first time

path-manager 2023-11-16 at 11 25 11

- You can maybe use [Form.FilePicker](https://developers.raycast.com/api-reference/user-interface/form#form.filepicker) and maybe here

path-manager 2023-11-16 at 11 25 38

- You can maybe use [getSelectedFinderItems](https://developers.raycast.com/api-reference/environment#getselectedfinderitems) for getting current path from Finder directly

- Add support for `~` in paths

path-manager 2023-11-16 at 11 27 44

- List Paths suffer from [Empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states)

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by erics118** [comment]
How does thei

**Comment by TNicko** [comment]
@erics118 Thanks for bringing my attention to Pins. Unfortunately I was not aware of it and it seems to have very similar functionality to my extension. The only difference would be that I can open up directories via a command with an alias rather than a key binding, but this is something Pins could easily add as an extra command/feature. 

If you have any ideas as to how i could extend the functionality to provide something useful and unique I'm all ears.

**Comment by pernielsentikaer** [comment]
So we can close this PR and maybe we can ask @SKaplanOfficial about you requests for Pins (that you or he can add)?

**Pull request closed without merging**